src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidRunCommandArguments.cs

```diff
@@ -77,15 +77,22 @@ protected override OptionSet GetTestCommandOptions() => new()
                 v => DeviceId = v
             },
             {
-                "boot-timeout=", "Timeout in seconds to wait for a device after boot competion",
-                v => {
-                    if (int.TryParse(v, out var number))
+                "launch-timeout=|lt=", "TimeSpan or number of seconds to wait for a device after boot competion",
```

Comment:
Same as the other file.

Can we pull this into some common ancestor or there cannot be one?

Reply:
Ancestor `TestCommandArguments` used for wasm without this parameter. Also, I think that this timeout makes no sense for `android uninstall`.